app/views/hourglass_ui/time_bookings/_list_entry_actions.slim

```diff
@@ -8,6 +8,6 @@
 = link_to '', start_hourglass_time_trackers_path(booking_params), class: 'icon-hourglass-continue js-hourglass-remote js-start-tracker', title: t('hourglass.ui.time_bookings.button_continue'), remote: true, method: 'post'
 
 - if policy(time_booking).change?
-  = link_to '', hourglass_ui_edit_time_bookings_path(time_booking), class: 'icon-edit js-show-inline-form', title: t(:button_edit), remote: true, data: {type: 'html'}
+  = link_to '', hourglass_ui_edit_time_bookings_path(time_booking), class: 'icon icon-edit js-show-inline-form', title: t(:button_edit), remote: true, data: {type: 'html'}
```

Comment:
For me changing the classes here does not make any difference in any of the default designs.
Do you use a custom style for Redmine?

Reply:
It seems everywhere else we used "icon icon-". - So great finding! Thanks!

Reply:
Sorry, I didn't test with the default themes this issue. You are right, we are using PurpleMine2 theme: https://github.com/mrliptontea/PurpleMine2/